### ISSUE TITLE
Failure when updating 0.13.15 -> 0.14.2

### ISSUE THREAD

**Issue by maxnordlund** [tree-sitter/tree-sitter#272]
Hi,

I just updated my email grammar to use the new rust based compiler, and get this error:

    /root/.tree-sitter/email.so: undefined symbol: tree_sitter_email

Full CI log: https://gitlab.com/maxnordlund/tree-sitter-email/-/jobs/159062493

Diff: https://gitlab.com/maxnordlund/tree-sitter-email/commit/b7491fce4df8428688f838aa6deeff37640b66b0

Browse source for the update: https://gitlab.com/maxnordlund/tree-sitter-email/tree/mn-update-tree-sitter-to-rust-version

<details>
<summary>grammar.js</summary>

```javascript
/* global grammar, seq, choice, repeat, repeat1, optional, prec, token, alias */
/* eslint no-unused-vars: off */
module.exports = grammar({
  name: "email_address",

  extras: $ => [$._newline],

  rules: {
    root: $ => seq(
      repeat($._newline),
      repeat1(seq(
        $.address,
        repeat1($._newline)
      ))
    ),

    _newline: $ => /\r?\n/,

    address: $ => seq(
      $.local_part,
      "@",
      $.domain,
    ),

    domain: $ => seq(
      optional($.comment),
      $._domain,
      optional($.comment),
    ),

    _domain: $ => seq(
      alias($.domain_labels, $.name),
      alias($.top_level_domain, $.top),
    ),

    domain_labels: $ => /(([a-z0-9]+(-[a-z0-9]+)?)+\.)+/,

    top_level_domain: $ => /[a-z]+/,

    local_part: $ => seq(
      optional($.comment),
      seq(
        $._local_sub_part,
        repeat(seq(".", $._local_sub_part)),
      ),
      optional($.comment),
    ),

    _local_sub_part: $ => choice(
      $._quote,
      $._valid_characters
    ),

    _valid_characters: $ => /(\\.|[-A-Za-z0-9!#$%&'*+/=?^_`{|}~])+/,

    _quote: $ => seq(
      '"',
      /(\\.|[^"])*/,
      '"',
    ),

    comment: $ => token(seq(
      "(",
      /[^)]*/,
      ")",
    )),
  },
})
```

</details>

**Comment by maxbrunsfeld** [comment]
Oh, oops. I think the new CLI is inferring the language name from the directory instead of from the ‘name’ grammar. I will fix this tomorrow.

**Comment by maxnordlund** [comment]
Lovely, thank you for all the hard work ❤️

**Comment by maxbrunsfeld** [comment]
I believe this is fixed in 0.14.4, which will be published shortly.
